test/unit/helpers/git.js

```diff
@@ -6,8 +6,11 @@ var sinon = require('sinon');
 var git = require('../../../helpers/git');
 var childProcess = require('child_process');
 
-describe('git', function () {
-	describe('has an error', function () {
+describe('getLastCommit', function () {
```

Comment:
We should describe with git, then encapsulate the rest inside of this `getLastCommit` describe block